deploy/charts/kubernetes-cluster-prep/sample-policy/cluster-authn-svc-def.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
You can delete the entire `sample-policy` directory. This will go in with the 'helm test'.